### PR TITLE
 addition of gan timer management

### PR DESCRIPTION
**Pull request by raphchta**
I didn't style the button

### CODE REVIEW

File: package-lock.json
Comment (Nogesma) [inline package-lock.json:1]: Remove this file, this is generated by npm. This project is using yarn.

File: package.json
Comment (Nogesma) [inline package.json:1]: For Svelte projects, all dependencies should be dev-dependencies. 
`yarn remove rxjs && yarn add -D rxjs`

File: src/tools/ganBluetooth.ts
Comment (Nogesma) [inline src/tools/ganBluetooth.ts:1]: I think this should be removed and the `gan-web-bluetooth` package should be used
`yarn add -D gan-web-bluetooth`

File: src/lib/Navbar.svelte
Comment (Nogesma) [inline src/lib/Navbar.svelte:1]: This does not need to be modified

File: src/routes/Solve.svelte
Comment (Nogesma) [inline src/routes/Solve.svelte:1]: This does not need to be modified

File: src/lib/Timer.svelte
Comment (Nogesma) [inline src/lib/Timer.svelte:1]: The timer connection should probably happen in the settings page. I will do it myself unless you want to.

**Review by Nogesma — CHANGES_REQUESTED** [review]
Thanks for the PR, I added some feedback.

You will also need to squash your commits before the merge can happen.

File: src/lib/Timer.svelte
Comment (raphchta) [reply src/lib/Timer.svelte:1]: no thank you for doing it

File: package.json
Comment (raphchta) [reply package.json:1]: if I do npm remove rxjs && npm add -D rxjs. his walk?

Comment (Nogesma) [reply package.json:1]: Yes, but please remove the package-lock.json generated by npm, and regenerate the yarn.lock by running `yarn`

**Pull request closed without merging**